dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowFactoryImpl.java

```diff
@@ -204,11 +206,11 @@ private Object convertMaptoObject(Map<String, Object> map, Class clazz) throws I
 		} else if (obj instanceof WorkflowActionClassParameter) {
 			return this.convertActionClassParameter(map);
 		} else if (obj instanceof WorkflowScheme) {
-			return this.convertScheme(map);
+			return WorkflowSchemeTransformer.transform(map);
```

Comment:
Again, why in this PR?

Reply:
Explained in the description also

Reply:
>Added DBTransformers for both WorkflowTask and WorkflowScheme instead of relying on BeanUtils.copyProperties , which was taking a performance hit, reported by Glowroot also